### ISSUE TITLE
Release latest version to npm

### ISSUE THREAD

**Issue by HarelM** [apache/cordova-plugin-file-transfer#294]
# Feature Request

## Motivation Behind Feature
In order to use this plugin one need to use a git url which is hard to follow in terms of versioning

## Feature Description
Release latest code to npm, last release was 3 year ago... :-(

## Alternatives or Workarounds
Use git url

**Comment by breautek** [comment]
When the plugin is ready to be released it will be released. Unfortunately I cannot give an ETA on when this will be.
